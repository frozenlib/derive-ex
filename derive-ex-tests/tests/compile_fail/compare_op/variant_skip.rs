use derive_ex::derive_ex;

#[derive_ex(PartialEq)]
enum X {
    #[partial_eq(ignore)]
    A,
}

fn main() {}
