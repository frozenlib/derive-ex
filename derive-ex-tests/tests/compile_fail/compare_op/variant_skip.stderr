error: cannot specify `skip` for enum variants
 --> tests/compile_fail/compare_op/variant_skip.rs:5:18
  |
5 |     #[partial_eq(ignore)]
  |                  ^^^^^^
